homeassistant/helpers/aiohttp_client.py

```diff
@@ -77,7 +77,7 @@ def async_get_clientsession(
 @bind_hass
 def async_create_clientsession(
     hass: HomeAssistant,
-    verify_ssl: bool = True,
+    ssl_context: bool | SSLContext = True
```

Comment:
This is a breaking change. Maybe keep the old parameter while adding the new parameter next to the old?

Reply:
That is a good idea, schould I log a warning when the old parameter is used that it will be deprecated?

Reply:
If we want to deprecate the old parameter, yes. I'm not sure if we want that. Let's hear what others think first.

Reply:
I agree with Martin.  We should keep both parameters here. See my comment above as well.